src/common/services/cortexApi.service.js

```diff
@@ -66,8 +65,8 @@ class CortexApi {
     return this.http(request);
   }
 
-  serializePath(path){
-    if(isArray(path)){
+  static serializePath( path){
```

Comment:
I like the idea of this being static (I think I've seen intellij suggest it) but I'm not sure how to test it as angular doesn't publish the class itself when using the injector, just the instance. Do you have any thoughts?